react/src/machine-coding/background-changer/Background.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import styles from './background.module.css';
+
+const Background = () => {
+
+  const ChangeBg = async () => {
+
+    // <-----------This is request with Axios----------->
+
+    // const options = {
+    //   method: 'GET',
+    //   url: 'https://colorful3.p.rapidapi.com/randomColor/2',
+    //   headers: {
+    //     'X-RapidAPI-Key': '4638530cfemshb5f9bea28be8a72p1d9bb8jsnef9a6f833608',
+    //     'X-RapidAPI-Host': 'colorful3.p.rapidapi.com'
+    //   }
+    // };
+
+    // try {
+    //   const response = await axios.request(options);
+    //   console.log(response.data[0].hex);
+    //   document.body.style.backgroundColor=`#${response.data[0].hex}`;
+    // } catch (error) {
+    //   console.error(error);
+    // }
+
+    // <-----------request with Axios Ends Here ----------->
+
+
+    const url = 'https://colorful3.p.rapidapi.com/randomNamedColor/2';
+    const options = {
+      method: 'GET',
+      headers: {
+        'X-RapidAPI-Key': '4638530cfemshb5f9bea28be8a72p1d9bb8jsnef9a6f833608',
+        'X-RapidAPI-Host': 'colorful3.p.rapidapi.com'
+      }
+    };
+
+    try {
+      const response = await fetch(url, options);
```

Comment:
for a new color to generate a normal function is enough. Do we need an API call to get a random color?

Reply:
I am using API for that so making API call . If you want I can simply Make an array and generate random color then but it will very bulky array so I am using api it generates random colour

Reply:
isn't it a simple function like below

```js
function getRandomColor() {
  var letters = '0123456789ABCDEF';
  var color = '#';
  for (var i = 0; i < 6; i++) {
    color += letters[Math.floor(Math.random() * 16)];
  }
  return color;
}
```

Reply:
Ok I Will add it this in fetch fucntion then